cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,24 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://conduit.mate.academy/user/login');
 
+    cy.get('h1')
+      .should('contain.text', 'Sign in');
+
+    // email entering
+    cy.get(':nth-child(1) > .form-control')
+      .type('bybeg@mailinator.com');
+
+    // password entering
+    cy.get(':nth-child(2) > .form-control')
+      .type('Pa$$w0rd!');
+
+    cy.get('.btn')
+      .click();
```

Comment:
It's better to use a more specific selector for the button, such as the button's text or a data-test attribute. This will make the test more robust and easier to understand.